### ISSUE TITLE
Handle Throttling Better

### ISSUE THREAD

**Issue by jantman** [jantman/awslimitchecker#435]
## Feature Request

When running in a GIANT account that's constantly near rate limits, CloudFormation DescribeStacks gets throttled and hits the max retries.

It would be helpful if awslimitchecker - including the Docker image - had a way to [configure](https://github.com/boto/boto3/issues/1571#issuecomment-395494622) the max retries both globally and per-service.

### Version

7.1.0

### Installation Method

Docker image

### Supporting Software Versions

Official docker image

### Actual Output

```
ClientError('An error occurred (Throttling) when calling the DescribeStacks operation (reached max retries: 4): Rate exceeded')
```

### Expected Output

No error.

**Comment by jantman** [comment]
This has been fixed by #445

**Comment by jantman** [comment]
This has been fixed in [8.0.0](https://github.com/jantman/awslimitchecker/releases/tag/8.0.0) which is now live on [PyPI](https://pypi.org/project/awslimitchecker/) and [Docker Hub](https://hub.docker.com/r/jantman/awslimitchecker/tags). Apologies for any delays in getting this released.
